_articles/download-data.md

```diff
@@ -0,0 +1,96 @@
+
+```
+synapse login
+```
+
+From Explore Data in the portal, select the **Download Options** icon and **Programmatic Options** to visualize the command to download the data subset.
+
+<img style="width: 25%;" src="/assets/images/programmatic-options-viz.png" alt="alt text">
+
+The command [`synapse get`](https://python-docs.synapse.org/build/html/CommandLineClient.html#get) with the `-q` argument downloads files from the entirety of the portal data that meet the specified condition. In this example, all *processed* and *metadata* files from the *MC-CAA* study will be downloaded. Execute the following command from the directory where you would like to store the files.
+
+```
+synapse get -q "SELECT * FROM syn11346063 WHERE ( ( "study" = 'MC-CAA' ) AND ( "dataSubtype" = 'processed' OR "dataSubtype" = 'metadata' ) )" 
+```
+
+Also in your working directory, you will find a *SYNAPSE_TABLE_QUERY_###.csv* file that lists the annotations associated with each downloaded file. Here, you might find helpful experimental details relevant to how the data was processed. Additionally, you will find important details about the file itself including the file version number.
+
+### R
+
+In order to download data programmatically with R, you need a list of synIDs that correspond to the files. For downloading a large set of files, we recommend using the Synapse Python client. The Python client has been optimized for multi-threaded download and will provide you with faster download speeds.
+
+Once you have identified the files you want to download from Explore Data, **Export Table** from Download Options.
+
+<img style="width: 25%;" src="/assets/images/export-table-viz.png" alt="alt text">
+
+You may choose to download the file as a *.csv* or *.tsv*. Files are named *Job-####*, where # includes a long set of numbers. Move this file to your working directory to proceed with the following steps. 
+
+[Install the Synapse R client](https://r-docs.synapse.org/#installation) `synapser` to download data from Synapse. [Login](https://r-docs.synapse.org/articles/manageSynapseCredentials.html#manage-synapse-credentials) to Synapse.
+
+```
+library(synapser)
+synLogin("my_username", "my_password")
+```
+Read the exported table into R, create a directory to store files and download data using `synGet`. If `downloadLocation` is not specified, the files are downloaded to a hidden directory called `/.synapseCache`. 
+```
+exported_table <- read.csv("Job-1342324092830598.csv")
+dir.create("files")
+lapply(exported_table$id, function(x) synGet(x,
+                                             downloadLocation = "./files"))
+```
+
+The `exported_table` also includes experimental details relevant to how the data was processed.
```

Comment:
This seems like a good place to use the term "annotations" to reinforce this vocabulary. 

Suggestion:
The `exported_table` also includes the file annotations containing both study details and experimental details relevant to how the data was processed.